tutorials/hello/hello_frontend/src/components/faucet.js

```diff
@@ -0,0 +1,135 @@
+import FixedMenuLayout from '../layout/header';
+import React, { useState } from "react";
+import { Form, Button, Divider, Container } from 'semantic-ui-react';
+import { cryptography, transactions } from '@liskhq/lisk-client/browser';
+import * as api from '../api';
+import { Buffer } from 'buffer';
+
+function Faucet() {
+    const [state, updateState] = useState({
+        address: '',
+        amount: '',
+        privateKey: '',
+        error: '',
+        transaction: {},
+        response: {}
+    });
+
+    const handleChange = (event) => {
+        const { name, value } = event.target;
+        updateState({
+            ...state,
+            [name]: value,
+        });
+    };
+
+    const handleSubmit = async (event) => {
+        event.preventDefault();
+
+        const client = await api.getClient();
+        const address = state.address;
+        const passphrase = 'weasel balance horse obtain love diary lesson reflect connect scheme decrease wrestle team sphere spring desert quote fever penalty rookie liquid harvest ride omit';
+        const privateKey = await cryptography.ed.getPrivateKeyFromPhraseAndPath(passphrase, "m/44'/134'/0'");
+        let responseError = '';
+        const signedTx = await client.transaction.create({
+            module: 'token',
+            command: 'transfer',
+
+            fee: BigInt(transactions.convertLSKToBeddows('0.01')),
+            params: {
+                tokenID: Buffer.from('0000000000000000', 'hex'),
+                amount: BigInt(transactions.convertLSKToBeddows(state.amount)),
+                recipientAddress: address,
+                accountInitializationFee: BigInt(transactions.convertLSKToBeddows('0.01')),
+                data: 'Please accept this grant from Lisk faucet.'
+            }
+        }, privateKey).catch(err => {
```

Comment:
```suggestion
        }, privateKey.toString('hex')).catch(err => {
```

this should fix the tx verification error :)

Reply:
The issue was with the versions that we were using, that has been corrected now.